critiquebrainz/frontend/views/bb_edition_group.py

```diff
@@ -0,0 +1,63 @@
+from flask import Blueprint, render_template, request
+from flask_login import current_user
+from werkzeug.exceptions import NotFound, BadRequest
+from flask_babel import gettext
+import critiquebrainz.db.review as db_review
+import critiquebrainz.frontend.external.bookbrainz_db.edition_group as bb_edition_group
+from critiquebrainz.frontend.forms.rate import RatingEditForm
+from critiquebrainz.frontend.views import get_avg_rating, EDITION_GROUP_REVIEWS_LIMIT
+
+
+edition_group_bp = Blueprint('edition_group', __name__)
+
+
+@edition_group_bp.route('/<uuid:id>')
+def entity(id):
+    id = str(id)
+    edition_group = bb_edition_group.get_edition_group_by_bbid(id)
```

Comment:
As I mentioned in https://github.com/metabrainz/critiquebrainz/pull/434#discussion_r915086575 here's a related part.
Take a look at [the documentation](https://github.com/wbolster/flask-uuid), by default it uses what's called "strict mode", so the `'/<uuid:id>'` in the URL will validate that it is only lower-case, and is a correctly formatted UUID otherwise it will return 404. So it looks like we're safe here!